py/examples/device_tapdetector_example/README.md

```diff
@@ -0,0 +1,31 @@
+
+Sample data is included in a .zip file to use. The following command/parameters works well for the sample data provided. Note that the sample data includes noise at the end.
+```py .\py\emotibit\device_tapdetector.py -sof .\emotibit5_AX.csv .\emotibit5_AY.csv .\emotibit5_AZ.csv -sod 3 -sot LocalTimestamp -soa AX AY AZ -stf .\cyton5.txt -std 3 -stt " Timestamp" -sta " Accel Channel 0" " Accel Channel 1" " Accel Channel 2" -n1 EmotiBit -n2 Cyton -o taps5 -w2 13 -h2 0.08```
+
+### Command Line Usage
+The tap detector is designed to be used with the command line. The tap detector is designed to work whether all dimensions of your data (e.g. X, Y, Z) are in one file or are in multiple files. The following example that is shown uses Emotibit files which store dimensions seperately and a cyton file where all dimensions are in one file:
+
+```$py device_tapdetector.py -sof .\emotibit_AX.csv .\emotibit_AY.csv .\emotibit_AZ.csv -sod 3 -sot LocalTimestamp -soa AX AY AZ -stf .\cyton.csv -std 3 -stt " Timestamp" -sta " Accel Channel 0" " Accel Channel 1" " Accel Channel 2" -tw1 0 10 -n1 EmotiBit -n2 Cyton -w1 1 -w2 13 -h1 0.1 -h2 0.4```
+
+In the above example, the three EmotiBit files are provided and correspondingly, 3 data columns are provided. Only one timestamp column is provided and it is expected to be the same across all files for source one. Additionally, one cyton file is provided and then the data columns are provided.
+
+**If you use more than one data input file, you must provide the explicit file for each data column.** This means that if you were to provide two files, such as ```.\emotibit_AX .\emotibit_AYZ``` you could not then specify three data columns such as ```AX AY AZ```. Instead you would need to write ```.\emotibit_AX .\emotibit_AYZ .\emotibit_AYZ``` and then provide the corresponding columns: ```AX AY AZ```. In other words, when more than one file is provided, the tap detector will assume that each file only provides one dimension of data.
+
+### Completing the Tapping Procedure:
+A prerequisite to using the tap detector is the proper collection of data. This section outlines the methods that should be use to collect the data.
+
+Data should be collected from two sources simultaneously. The following is a set of generic steps to take for data collection of any two devices to use in the tap detector, you may need to take additional steps depending on the devices you are using and the best practices for using those devices.
+
+1. **Setup your Environment(s)** - Ensure that you have a working environment for both devices you plan to use that allows you to record data.
```

Comment:
See note above about heading structure, but i think using the word "environment" when dealing with python can add ambiguity. Maybe stash this under a "device setup" heading?

Reply:
Good point, changed.